.github/workflows/ci.yml

```diff
@@ -5,9 +5,9 @@ name: Node.js CI
 
 on:
   push:
-    branches: [ master, beta ]
+    branches: [ master, beta, Release2.7 ]
```

Comment:
Branch Release2.7 is for the 2.7.x release (to also fix IE8 for the 2.7.4 with a 2.7.5 release)